cn.navclub.vdbfx.json

```diff
@@ -0,0 +1,77 @@
+{
+    "app-id" : "cn.navclub.dbfx",
+    "runtime" : "org.gnome.Sdk",
+    "runtime-version" : "42",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "v-dbfx",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "/share/vala",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name" : "libgee",
+            "buildsystem" : "autotools",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://gitlab.gnome.org/GNOME/libgee.git"
+                }
+            ]
+        },
+        {
+            "name" : "flatpak_mysql",
+            "no-autogen" : true,
+            "buildsystem" : "cmake",
+            "cleanup" : [
+                "lib",
+                "include"
+            ],
+            "sources" : [
+                {
+                    "type" : "git",
+                    "branch" : "main",
+                    "url" : "https://github.com/GZYangKui/flatpak-mysql.git"
+                }
+            ]
+        },
+        {
+            "name" : "dbfx",
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "branch" : "main",
+                    "url" : "https://github.com/GZYangKui/databasefx/v-dbfx.git"
+                }
+            ],
+            "config-opts" : [
+                "gsettings",
+                "set",
+                "org.gtk.Settings.Debug",
+                "enable-inspector-keybinding",
+                "true"
+            ]
+        }
+    ],
+    "build-options" : {
```

Comment:
this is usually above modules....

also I'm not sure this is does what you think it does. `GDK_GL` is a runtime variable for `GDK`. But maybe I missed something.